Nos/Views/SideMenuContent.swift

```diff
@@ -47,7 +47,8 @@ struct SideMenuContent: View {
                         if let name = currentUser.author?.safeName {
                             Text(name)
                                 .foregroundColor(.primaryTxt)
-                                .font(.clarity(.bold, textStyle: .title2))
+                                .dynamicTypeSize(...DynamicTypeSize.accessibility2)
+                                .font(.clarityBold(baseSize: 24, relativeTo: .title2))
```

Comment:
I wish this could be:
```suggestion
                                .font(.clarityBold(.title2))
```

Is that not possible? In trying it out myself, it seems to work though we may want to limit the dynamic type size to accessibility1 or even xxxLarge.